experimental/learn_descriptors/learn_descriptors.cc

```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+
+#include "learn_descriptors.hh"
+
+void robot::experimental::learn_descriptors::hello_world(const std::string &in)
```

Comment:
Make the namespace separate.

Reply:
done in commit 6d07ea92c4fc41e38ce5a3523b2719ab84e8a9dc